frontend/src/components/common/Filter/components/FilterContent/components/FilterViews/components/FilterSinglePanelCategoryView/index.tsx

```diff
@@ -29,10 +30,11 @@ function filterViews(
 ): SelectCategoryValueView[] {
   if (searchValue) {
     return views.filter(({ label }) =>
-      label.toLowerCase().includes(searchValue)
+      label.toLowerCase().includes(searchValue.toLowerCase())
```

Comment:
just curious, was this a bug that wasn't caught before or a new behavior?

Thank you!

Reply:
This is sneaking in a fix for a bug. I will create a new ticket and make it its own PR.

Reply:
Reverted.

Reply:
Thanks so much, @NoopDog ! 

Sorry I didn't mean to ask you to create a ticket and a fix in a separate PR, since that seems to be a higher cost than the fix itself in this case 😆 Was just genuinely curious what the code change was for lol 

But I do appreciate you taking the time to do it, since technically it's probably the best practice to have focused PRs!